### PR TITLE
Don't allow horizontal scrolling of mobile menu (bug 1161478)

### PR DESCRIPTION
**Pull request by mstriemer**
https://bugzilla.mozilla.org/show_bug.cgi?id=1161478

### CODE REVIEW

File: src/media/css/elements--nav.styl [hunk 1]
```diff
@@ -109,6 +109,7 @@ mkt-nav {
     font-size: 18px;
     height: 100%;
     left: -1 * $nav--width;
+    overflow-x: hidden;
     overflow-y: scroll;
```
Comment (chuckharmston) [inline src/media/css/elements--nav.styl]: Should this be `overflow-y: auto` so as not to force a visible scrollbar when it's not necessary on Windows?

**Comment by ngokevin** [comment]
r+

**Comment by chuckharmston** [comment]
r+

**Comment by ngokevin** [comment]
early reviewer gets the r+

File: src/media/css/elements--nav.styl (hunk 1 above)
Comment (mstriemer) [inline src/media/css/elements--nav.styl]: Still got my Windows VM open, I'll test it out.

Comment (mstriemer) [inline src/media/css/elements--nav.styl]: screenshot 2015-05-05 13 33 13

**Comment by mstriemer** [comment]
Switched both of the `overflow-y: scroll;`s to `overflow-y: auto;`.

menu-windows mov

**Comment by chuckharmston** [comment]
Love it, nicely done.

**Comment by mstriemer** [comment]
Good catch! Thanks.

**Comment by chuckharmston** [comment]
Would also be nice if we could do something like this to prevent the top of the scrollbar from being clipped by the back button:

    mkt-nav {
        overflow-y: hidden;
    } 
    mkt-nav-child {
        margin-top: 60px;
        padding: 0 10px;
    }

Rely on the scroll to happen on `mkt-nav-root` and `mkt-nav-child` instead.

**Pull request merged**